iothub/device/src/DirectMethod/DirectMethodResponse.cs

```diff
@@ -21,24 +25,36 @@ public DirectMethodResponse(int status)
         /// <summary>
         /// The status of direct method response.
         /// </summary>
+        [JsonProperty]
```

Comment:
q - it looks like this PR is adding the json annotations - how did it work previously? Were the fields being explicitly set, as opposed to being set during deserialization?

Reply:
if this is on the device side, it may not change anything. I'll get rid of them and see what happens.